src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return (
+    sourceString
```

Comment:
The code is not following the good example for chained method calls provided in the checklist. Each method call should be on a new line with the same indentation for better readability.